web-client/src/components/Buttons/index.tsx

```diff
@@ -0,0 +1,31 @@
+import { Button } from 'antd';
```

Comment:
I support the creation of a reusable component library.  This buttons file is a start.